docs/data/upload_ui.md

```
@@ -34,4 +34,22 @@ In particular, the following file formats are supported for uploading (and conve
 - [N5 datasets](./n5.md)
 
+We support a variety of data types for the uploaded data. To make sure that your data can be uploaded to WEBKNOSSOS take a look into this table of supported data types for color and segmentation layers:
+
+|   dtype    | Color Layers  |       Segmentation Layers   |
+|------------|------------|------------|
+|   uint8    |       ✓     |   ✓                       |
+|   uint16   |       ✓     |   ✓                       |
+|   uint24  rgb  |       ✓   |   does not apply        |
+|   uint32   |       ✓     |   ✓                       |
+|   uint64   |       ✗     |   (✓)  [(til 2⁵³−1)](https://github.com/scalableminds/webknossos/issues/6921)          |
+|   |  |  |  |
```

markdownlint-cli2 (0.17.2)

Table column count
Expected: 3; Actual: 4; Too many cells, extra data will be missing

(MD056, table-column-count)